content/src/service/ServiceImpl.ts

```diff
@@ -341,51 +350,14 @@ export class ServiceImpl implements MetaverseContentService {
   }
 
   async getEntitiesByIds(ids: EntityId[]): Promise<Entity[]> {
```

Comment:
Delete this method and use the activeEntities component directly from the controller